test/e2e/tinkhwcount.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"sigs.k8s.io/yaml"
+)
+
+// TinkTest maps each Tinkbell test with the hardware count needed for the test.
```

Comment:
nit: Let's use Tinkerbell everywhere instead of Tink
```suggestion
// TinkerbellTest maps each Tinkerbell test with the hardware count needed for the test.
```